src/test/java/core/basesyntax/SupplyOperationTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ServiceShopForTestsException;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.ServiceShopForTests;
+import core.basesyntax.service.operation.ServiceShopForTestsImpl;
+import core.basesyntax.service.operation.SupplyOperation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class SupplyOperationTest {
+    private ServiceShopForTests serviceShopForTests = new ServiceShopForTestsImpl();
```

Comment:
please remove this. You are not testing SupplyOperationTest in this class. You should test SupplyOperation without using SupplyOperationTest